golem.yaml

```diff
@@ -0,0 +1,87 @@
+package:
+  name: golem
+  version: "1.2.0"
+  epoch: 0
+  description: "Golem is an open source durable computing platform that makes it easy to build and deploy highly reliable distributed systems."
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - gcc
+      - openssl-dev
+      - pkgconf
+      - protobuf-dev
+      - protoc
+      - rust
+      - rustup
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/golemcloud/golem
+      tag: v${{package.version}}
+      expected-commit: 51d5cbb1b9fcca357561c32577998ef74ce30f3d
+
+  - name: Configure and build
+    runs: |
+      # rustup install stable
+      # rustup target add wasm32-wasip1
+      # rustup target add wasm32-unknown-unknown
+
+      # # Put our rustup-installed toolchains into $PATH
+      # export PATH="$(dirname "$(rustup which cargo)"):$PATH"
+
+      export PATH=$PATH:$HOME/.cargo/bin
+
+      cargo install --force cargo-make
+
+      cargo make --profile ci build-release
+
+      mkdir -p ${{targets.contextdir}}/usr/bin/
+      mv ./target/release/golem-component-service ${{targets.contextdir}}/usr/bin/golem-component-service
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-component-service
```

Comment:
Should we have runtime deps here? https://github.com/golemcloud/golem/blob/0a04f54f954ec859b59b810d9184cadba3fafaec/golem-component-service/docker/Dockerfile#L20C42-L20C79

Reply:
this is a fair concern, let me add them, thanks for the catch!

Reply:
but then I noticed that its using a ubuntu as base image https://github.com/golemcloud/golem/blob/0a04f54f954ec859b59b810d9184cadba3fafaec/golem-component-service/docker/Dockerfile#L1 so there are lots of utilities included into the image so we to have make sure that which are needed for runtime functionality so the only of knowing this is to create an image from it and test it at runtime so I think we might leave runtime dependencies empty at this moment.

Reply:
yeah actually i have never seen a use-case why we add pkgconf at the runtime deps, and why they add `libssl-dev` to runtime? Looking at the [Cargo](https://github.com/golemcloud/golem/blob/0a04f54f954ec859b59b810d9184cadba3fafaec/golem-component-service/Cargo.toml) file, we may need to have `libssl`, `ca-certificates`? at the runtime anyway since there is gRPC dependency.

Reply:
`cargo tree -i openssl` it seems some deps has transitive dependency on openssl, and since `reqwest` [doesn't use](https://github.com/golemcloud/golem/blob/0a04f54f954ec859b59b810d9184cadba3fafaec/Cargo.toml#L163) the `rustls` feature, let's have the `pkgconf`, `libssl-dev`, and `ca-certificates` packages in the runtime to match with upstreams.